src/views/product-landing/helpers/get-inline-content-maps.ts

```diff
@@ -0,0 +1,31 @@
+import gatherUniqueValues from 'lib/gather-unique-values'
+import {
+  getInlineCollections,
+  getInlineTutorials,
+  InlineCollections,
+  InlineTutorials,
+} from 'views/product-tutorials-view/helpers/get-inline-content'
+
+export async function getInlineContentMaps(data: unknown): Promise<{
```

Comment:
This new helper accepts some arbitrary `data`, traverses it, and returns `{ inlineTutorials, inlineCollections }`. Used to transform raw content (slugs) into tutorial & collection card data server-side.